### PR TITLE
Added recipes for seamm-datastore and flask-authorize

### PR DESCRIPTION
**Pull request by paulsaxe**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/seamm-datastore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+test:
+  imports:
+    - seamm_datastore
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/molssi-seamm/seamm-datastore
```
Comment (synapticarbors) [inline recipes/seamm-datastore/meta.yaml:39]: This link appears to be incorrect. Should be https://github.com/molssi-seamm/seamm_datastore

File: recipes/seamm-datastore/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools
+  run:
```
Comment (synapticarbors) [inline recipes/seamm-datastore/meta.yaml:23]: The package appears to do a poor job at specifying dependencies, but is `marshmallow_sqlalchemy` also required?

**Review by synapticarbors — COMMENTED** [review]
Just a few comments. Also make sure you're aware of the following about putting inter-dependent packages in the same PR (see note):
https://conda-forge.org/docs/maintainer/adding_pkgs.html#adding-multiple-packages-at-once

File: recipes/seamm-datastore/meta.yaml (hunk 2 above)
Comment (paulsaxe) [reply recipes/seamm-datastore/meta.yaml:23]: Yes, along with several other packages as you noted. I believe the requirements are now correct and have tested installing from PyPi in a clean conda environment. At least all the tests work.

File: recipes/seamm-datastore/meta.yaml (hunk 1 above)
Comment (paulsaxe) [reply recipes/seamm-datastore/meta.yaml]: Thanks! Dashes and underscores will be the death of me! The PyPi links are particularly fun in this regard.

**Pull request merged**